packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1434,7 +1434,9 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
 
     removed.forEach((participant) => {
       const subscriber = this.participantSubscribers.get(toFlatCommunicationIdentifier(participant.identifier));
-      subscriber && subscriber.unsubscribeAll();
+      if (subscriber) {
```

Comment:
For these use null propagation
```suggestion
      subscriber?.unsubscribeAll();
```